keyboards/rabbit/rabbit68/rules.mk

```diff
@@ -0,0 +1,59 @@
+# MCU name
+MCU = atmega32u4
+
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   atmega32a    bootloadHID
+BOOTLOADER = halfkay
+
+
+# If you don't know the bootloader type, then you can specify the
```

Comment:
Lines 15 to 38 can be removed; QMK defines these as defaults through `MCU` and `BOOTLOADER`.